pkg/constants/constants.go

```diff
@@ -21,4 +21,6 @@ const (
 	WorkloadMode = "workload"
 
 	XDP_PROG_NAME = "xdp_shutdown"
+
+	RootCertPath = "/var/run/secrets/istio/root-cert.pem"
```

Comment:
can we also remove that same one from nets/connect.go

Reply:
ok